performance/scripts/docker_stats_charts.nim

```diff
@@ -0,0 +1,107 @@
+import algorithm, sequtils, strformat, strutils, tables
+import ../types
+import ./common
+
+const CHARTS_DATA = [
+  ResourceChartData(title: "CPU %", yAxis: "CPU %", chartType: ResourceChartType.Cpu),
+  ResourceChartData(
+    title: "Memory Usage (MB)",
+    yAxis: "Memory (MB)",
+    chartType: ResourceChartType.Memory,
+  ),
+  ResourceChartData(
+    title: "Network Throughput (MB/s)",
+    yAxis: "MB/s",
+    chartType: ResourceChartType.NetThroughput,
+  ),
+  ResourceChartData(
+    title: "Total Network Data (MB)", yAxis: "MB", chartType: ResourceChartType.NetTotal
+  ),
+]
+
+proc readCsv(path: string): seq[ResourceChartsSample] =
+  let lines = readFile(path).splitLines()
+  var data: seq[ResourceChartsSample]
+  for i, line in lines:
+    if i == 0 or line.len == 0:
```

Comment:
we should skip every line that has `len == 0` AND the first line with non-zero `len` (not necessarily `i == 0`)